frontend/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:16
+WORKDIR /app
+COPY package.json ./
+RUN npm install
+COPY . .
+EXPOSE 3000
+CMD ["npm", "start"]
```

Comment:
Why are we not using `npm run build`? I might be wrong but I think `npm start` is for development while `npm run build` creates a static build file which is smaller in size and optimized for production

Reply:
Indeed. Let's use `npm run build` then.

Reply:
Let me look into this. It seems it's not as straightforward as just changing the command. Will update again.

Reply:
Let's keep the scope of this PR to development. Most likely we will have another `Dockerfile` for production.